askbot/models/__init__.py

```diff
@@ -4109,6 +4113,11 @@ def record_spam_rejection(
     post_anonymous_askbot_content,
     dispatch_uid='post_anon_content_on_login'
 )
+signals.post_save.connect(
```

Comment:
This may not be necessary as hopefully we call `clear_cached_data` or `reset_cached_data` when necessary.

NB: reset_cached_data is more expensive (actually more expensive than .save() ) and better not be called many times per request-response cycle, I don't think it actually is, but something to keep in mind.

clear_cached_data is cheap, just invalidates cache.